daisy/step_copy_gcs_objects_test.go

```diff
@@ -56,10 +56,10 @@ func TestCopyGCSObjectsValidate(t *testing.T) {
 		} else if m == "GET" && u == "/b/bucket2?alt=json&prettyPrint=false&projection=full" {
 			fmt.Fprint(w, `{}`)
 		} else if m == "POST" && (u == "/b/bucket1/o?alt=json&prettyPrint=false&projection=full&uploadType=multipart" ||
-			u == "/upload/storage/v1/b/bucket1/o?alt=json&prettyPrint=false&projection=full&uploadType=multipart") {
+			u == "/upload/storage/v1/b/bucket1/o?alt=json&name=daisy-validate--abcdef&prettyPrint=false&projection=full&uploadType=multipart") {
```

Comment:
This doesn't seem related to the  feature implemented in this PR. Why this change?

Reply:
Same as above, when the versions were updated in go.mod for the imported libraries these tests broke with name being added to the expected URL output.